gcc/rust/hir/tree/rust-hir.cc

```diff
@@ -1047,6 +1047,33 @@ BlockExpr::as_string () const
   return str;
 }
 
+std::string
+AnonConst::as_string () const
+{
+  std::string istr = indent_spaces (enter);
```

Comment:
Could be replaced with a string_stream

Reply:
there are no string streams in any other implementations, I think it'd be better to fix them all at once